src/TraceEvent/Stacks/Histogram.cs

```diff
@@ -181,11 +181,11 @@ private IEnumerable<float> GetEnumerable()
         /// </summary>
         private readonly HistogramController m_controller;
 
-        private float[] m_buckets;               // If null means is its single value or no values
```

Comment:
As mentioned in the general comments, For histograms as well as CallTreeNodeBase  (anything that is a substantial part of a CallTreeNode), I would like this to be under #ifdef control (currently defaulted to float), so we can experiment / measure and see if we can't get BOTH size and accuracy.